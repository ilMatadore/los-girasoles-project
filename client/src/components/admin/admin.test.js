import React from 'react';
import { shallow } from 'enzyme';
import Admin from './admin.component';

it('Expect to render Card component', () => {
    expect(shallow(<Admin />)).toMatchSnapshot();
})