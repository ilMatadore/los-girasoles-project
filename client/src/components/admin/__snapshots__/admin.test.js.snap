// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Card component 1`] = `
<div
  className="admin-panel-container"
>
  <h1>
    Admin Panel
  </h1>
  <div
    className="admin-panel-contentAndOptions-container"
  >
    <div
      className="admin-panel-options-container"
    >
      <div>
        Users
      </div>
      <div>
        Products
      </div>
      <div>
        Canastas
      </div>
      <div>
        Adicionales
      </div>
    </div>
    <div
      className="admin-panel-content-container"
    >
      Content
    </div>
  </div>
</div>
`;
